src/PeriodEnd/PaymentsDue/DeployPaymentsDue/sql/dml/13 PeriodEnd.PaymentsDue.PreRun.Staging.ApprenticeshipEarnings2.sql

```diff
@@ -21,14 +21,14 @@ SELECT
     ade.ApprenticeshipContractType,
```

Comment:
Nothing wrong exactly with the change, but the SQL script in general is very inefficient. The query plan shows table scans, filtering, hash matches and sorts that could be rationalised by rewriting the query.